Sources/ClientRuntime/Networking/Http/HttpTelemetry.swift

```diff
@@ -10,7 +10,7 @@ import struct Smithy.AttributeKey
 /// Container for HTTPClient telemetry, including configurable attributes and names.
 ///
 /// Note: This is intended to be used within generated code, not directly.
-public class HttpTelemetry {
+public actor HttpTelemetry {
```

Comment:
`HttpTelemetry` is now an actor, which enforces exclusive access to its internal data.